api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -320,7 +320,7 @@ public static <T extends CustomResource> List<String> apiVersions(Class<T> cls)
 
             List<String> versions;
             try {
-                versions = singletonList(group + "/" + (String) cls.getField("VERSION").get(null));
+                versions = singletonList(group + "/" + cls.getField("VERSION").get(null));
```

Comment:
Stupid question - what is the point of `get(null)`?

Reply:
TBH, I have no idea.

Reply:
Fair enough.